### DISCUSSION TITLE
500 Error with blank white page

### DISCUSSION THREAD

**Discussion by N-Yost** [mastodon/mastodon#23193] [Q&A]
As the title says, I'm trying to setup a server and I can't get past this issue. I have followed the how to step by step with a clean install on both Ubuntu 20.04 LTS and 22.10 on a Linode VPS. I'm new to Nginx and not really sure how to proceed after looking around for hours for an answer. Any pointers as to what to look into would be really appreciated.

Nginx error log:
`2023/01/22 03:40:35 [alert] 606#606: *69 open socket #16 left in connection 9
2023/01/22 03:40:35 [alert] 606#606: *70 open socket #15 left in connection 10
2023/01/22 03:40:35 [alert] 606#606: *84 open socket #19 left in connection 12
2023/01/22 03:40:35 [alert] 606#606: aborting`

access log:
`77.237.190.205 - - [22/Jan/2023:03:54:54 +0000] "POST / HTTP/2.0" 499 0 "-" "Go-http-client/2.0"
103.133.242.202 - - [22/Jan/2023:03:55:03 +0000] "POST / HTTP/2.0" 403 0 "-" "Go-http-client/2.0"
76.125.160.47 - - [22/Jan/2023:03:55:04 +0000] "GET / HTTP/2.0" 500 0 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:108.0) Gecko/20100101 Firefox/108.0"
136.169.169.51 - - [22/Jan/2023:03:56:09 +0000] "POST / HTTP/2.0" 403 0 "-" "Go-http-client/2.0"`

**Comment by mtbluedog** [comment]
Which "how to step by step" did you follow?
Take a look at https://stackoverflow.com/search?q=open+socket+left+in+connection+nginx

**Reply by N-Yost** [reply]
I followed the pre-setup here:
https://docs.joinmastodon.org/admin/prerequisites/

and then the actual install here:
https://docs.joinmastodon.org/admin/install/

I'm not sure if the left in connection is currently going on. Checking the error log after trying it again is not showing any new error.

**Comment by ClearlyClaire** [comment]
Please look at Mastodon's logs (`journalctl -u mastodon-web`) rather than nginx when encountering a 500 error
